mappings/net/minecraft/client/util/TextureUtils.mapping

```diff
@@ -2,44 +2,109 @@ CLASS none/bxg net/minecraft/client/util/TextureUtils
 	FIELD a TEXTURE_MISSING Lnone/bwt;
 	FIELD b TEXTURE_MISSING_DATA [I
 	FIELD c LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD d textureDataBuf Ljava/nio/IntBuffer;
+	FIELD e GAMMA_RAMP [F
+	FIELD f colorBlurBuffer [I
 	METHOD a createTextureId ()I
 	METHOD a destroyTextureId (I)V
 		ARG 0 id
-	METHOD a (II[[I)[[I
-	METHOD a (ILjava/awt/image/BufferedImage;)I
+	METHOD a allocTextureStorage (III)V
```

Comment:
allocate. We generally do not shorten method names unless the resulting name would be absurd.